java/fory-core/src/main/java/org/apache/fory/builder/BaseObjectCodecBuilder.java

```diff
@@ -240,6 +240,9 @@ public String genCode() {
     ctx.addField(ctx.type(Fory.class), FORY_NAME);
     Expression encodeExpr = buildEncodeExpression();
     Expression decodeExpr = buildDecodeExpression();
+
+    Expression xlangEncodeExpr = buildXlangEncodeExpression();
```

Comment:
we should do this only fury is configured to use xlang mode, and when using xlang mode, do not generate code for java native serialization, i.e. don't generate code for `read/write`